src/TimeSeries.svelte

```diff
@@ -635,17 +642,29 @@
   }
 
   onMount(async () => {
+    regionalTrends.subscribe((regionalTrends_store) => {
+      regionalTrendsByPlaceId = regionalTrends_store;
+      if (placeId && regionsByPlaceId.size > 0 && regionalTrendsByPlaceId.size > 0 && chartContainerElement) {
+        isChartReady = true;
+        generateChart();
+      }
+      else {
+        isChartReady = false;
+      }
+    });
     params.subscribe((param) => {
       placeId = param.placeId;
-      if (placeId && regionsByPlaceId && regionalTrendsByPlaceId && chartContainerElement) {
+      if (placeId && regionsByPlaceId.size > 0 && regionalTrendsByPlaceId.size > 0 && chartContainerElement) {
```

Comment:
using stores now instead of just passing the data. this allows us to update the content when we receive the zipcodes. because stores always "exist" we need to check for the size to ensure content is there.